packages/extension/src/manager/contexts-manager.ts

```diff
@@ -460,4 +464,43 @@ export class ContextsManager {
   protected handleStatus(_status: V1Status): void {
     // TODO: https://github.com/podman-desktop/extension-kubernetes-dashboard/issues/103
   }
+
+  async bulkDeleteObjects(objects: { kind: string; name: string; namespace?: string }[]): Promise<void> {
+    const message = this.getTextualObjectsList(objects);
+    const result = await window.showInformationMessage(`Are you sure you want to delete ${message}?`, 'Yes', 'Cancel');
+    if (result !== 'Yes') {
+      return;
+    }
+    for (const object of objects) {
```

Comment:
We are stopping at the first error maybe a better error management would be good, no ?